### PR TITLE
feat: warn when redefine plugin

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] commit message follows commit guidelines

##### Affected core subsystem(s)

pluginLoader

##### Description of change

Closes eggjs/egg#125

### CODE REVIEW

File: lib/loader/mixin/plugin.js [hunk 1]
```diff
@@ -324,29 +324,34 @@ module.exports = {
...
-        continue;
+    for (const name in plugins) {
+      const plugin = plugins[name];
+      let targetPlugin = target[name];
+      if (!targetPlugin) {
+        targetPlugin = target[name] = {};
       }
-      if (target[name][prop] && Array.isArray(plugin[prop]) && !plugin[prop].length) {
-        continue;
+      if ((targetPlugin.path || targetPlugin.package) && (plugin.path || plugin.package)) {
```
Comment (popomore) [inline lib/loader/mixin/plugin.js]: 就加了这个判断

**Comment by fengmk2** [comment]
+1

**Comment by popomore** [comment]
我发个 1.0

**Comment by popomore** [comment]
+ egg-core@1.0.0

**Pull request merged**